transports/daily/package.json

```diff
@@ -21,13 +21,13 @@
     "lint": "eslint . --ext ts --report-unused-disable-directives --max-warnings 0"
   },
   "devDependencies": {
-    "@pipecat-ai/client-js": "^0.4.0",
+    "@pipecat-ai/client-js": "file:../../../pipecat-client-web/client-js",
```

Comment:
Just adding this comment here as a bookmark/reminder to update this once the new client-js is released